tests/install-site-phpcode.test.js

```diff
@@ -84,8 +104,16 @@ describe('install-site.phpcode', () => {
         expect(stdOut.shift().trim()).toStrictEqual('{"message":"Beginning install tasks","type":"install"}')
         expect(stdOut.pop().trim()).toStrictEqual('{"message":"Performing install task (12 \\/ 12)","type":"install"}')
         assertOutput(stdErr, '')
+        stdOut.length = 0
+
+        await runPhpCode(php, rootPath + '/public/assets/login-admin.phpcode')
+        const loginOutput = JSON.parse(stdOut.join('').trim());
+        console.log(loginOutput)
 
         const [cgiOut, cgiErr, phpCgi] = createCgiPhp({ configFixturePath, persistFixturePath });
+        phpCgi.cookies.set(loginOutput.params.name, loginOutput.params.id)
+
+        // @todo ping a debug script to check on HTTP_COOKIES, user isn't logged in.
         await phpCgi.request({
```

Comment:
Add some kind of debug script into `tests/fixtures/drupal-core` to debug cookies and sessions